Source/PluginEditor.cpp

```diff
@@ -15,25 +15,59 @@ SubsynthAudioProcessorEditor::SubsynthAudioProcessorEditor (SubsynthAudioProcess
 {
     // Make sure that before the constructor has finished, you've set the
     // editor's size to whatever you need it to be.
-    setSize (1000, 300);
+    ///setSize (1000, 300);
     setupGain();
     
+    setSize (800, 565);
+
     waveSelect.addItem("Sine", 1);
     waveSelect.addItem("Square", 2);
     waveSelect.addItem("Saw", 3);
     waveSelect.addItem("Triangle", 4);
     waveSelect.setSelectedId(1);
-        
+            
+    filterSelect.addItem("Low Pass", 1);
+    filterSelect.addItem("Band Pass", 2);
+    filterSelect.addItem("High Pass", 3);
+    filterSelect.onChange = [this] { filterChanged(); };
+    filterSelect.setSelectedId(1);
+
+    filterCutoff.setSliderStyle(juce::Slider::SliderStyle::LinearHorizontal);
+    filterCutoff.setRange(20.0f, 20000.0f, 0.01f);
```

Comment:
Sorry 1 other very tiny change. I don't think we need this much resolution. Filter setting down to 1 Hz is probably better than showing the float.
```suggestion
    filterCutoff.setRange(20.0f, 20000.0f, 1.0f);
```